main.tf

```diff
@@ -0,0 +1,36 @@
+data "aws_caller_identity" "default" {}
+
+module "label" {
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.5.3"
```

Comment:
better to use https://github.com/cloudposse/terraform-terraform-label.
Latest versions of `terraform-null-label` are causing issues with counts in different top-level modules